backend/variants/views/api/export.py

```
@@ -0,0 +1,1404 @@
+            add_genes_regions_settings_table(doc, data)
+        elif "Other Settings" in title:
+            add_other_settings_table(doc, data)
+        elif "Prioritization" in title:
+            add_prioritization_settings_table(doc, data)
+        else:
+            # Default formatting for other sections
+            add_generic_section(doc, data)
+
+        # Add some spacing
+        doc.add_paragraph()
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)